emukit/core/optimization/acquisition_optimizer.py

```diff
@@ -10,19 +10,22 @@ def __init__(self, space: ParameterSpace, **kwargs) -> None:
         self.gpyopt_space = space.convert_to_gpyopt_design_space()
         self.gpyopt_acquisition_optimizer = GPyOpt.optimization.AcquisitionOptimizer(self.gpyopt_space, **kwargs)
 
-    def optimize(self, acquisition: Acquisition):
+    def optimize(self, acquisition: Acquisition, context: dict = None):
```

Comment:
No need to do anything for this PR, but we should take care of #15 and refactor this. I've marked it with 1.0